src/kixi/heimdall/service.clj

```diff
@@ -161,3 +161,26 @@
                               {:user-id user-id
                                :group-id group-id}) true)
       false)))
+
+(defn remove-member
+  [session {:keys [user-id group-id]}]
+  (let [user-id  (java.util.UUID/fromString user-id)
+        group-id (java.util.UUID/fromString group-id)]
+    (member/remove-member session user-id group-id)
+    {:kixi.comms.event/key :kixi.heimdall/member-removed
```

Comment:
Similarly to other comments, you don't want to be returning an event here, I don't think